packages/@expo/cli/e2e/fixtures/with-blank/app.json

```diff
@@ -1,5 +1,6 @@
 {
     "expo": {
+        "runtimeVersion": "1.0",
```

Comment:
@EvanBacon - this is the part of this PR that would be good for you to review. The template has runtime version in it, so this test now requires runtimeVersion in the fixture. (can no longer use expo-updates without specifying runtime version)